src/Attributes.elm

```diff
@@ -178,7 +178,7 @@ getDescription attribute value =
     in
         (List.filter (isLessThanAttribute value) attributeDescriptions)
             |> List.head
-            |> Maybe.map snd
+            |> Maybe.map Tuple.second
```

Comment:
Maybe. Why not just use records for everything ;-) ? :-) "second" means nothing apart from "take the second element of this anonymous data structure". That's not very informative... I have to go hunting for what that means.